src/reducers/movieList.js

```diff
@@ -0,0 +1,47 @@
+import {ADD_MOVIE, DELETE_MOVIE, EDIT_MOVIE, GET_MOVIE_LIST} from "../actions";
+import {MOVIES} from "../mock/MOCK";
+
+let nextMovieId = 0;
+
+const movieList = (state = [], action) => {
+    switch (action.type) {
+        case GET_MOVIE_LIST:
+            return MOVIES.map(item => {
+                return {...item, id: nextMovieId++}
```

Comment:
reducers are pure functions. this means it's strictly forbidden to use side effect in a reduce
https://redux.js.org/basics/reducers#handling-actions